docs/static/ds6/skin.css

```diff
@@ -480,13 +517,15 @@ span.fake-menu__status {
 }
 .menu__items--grow[role="menu"],
 .menu__items--grow-reverse[role="menu"],
+.listbox__options--grow[role="listbox"],
+.listbox__options--grow-reverse[role="listbox"],
 .fake-menu__items--grow,
 .fake-menu__items--grow-reverse {
   min-width: calc(100%);
   width: auto;
-  max-width: calc(150%);
```

Comment:
Was there a reason for removing this?

Reply:
Yes, it was not allowing the grow to ... grow.